Doc/library/gzip.rst

```diff
@@ -120,9 +120,8 @@ The module defines the following items:
    .. method:: peek(n)
 
       Read *n* uncompressed bytes without advancing the file position.
-      At most one single read on the compressed stream is done to satisfy
-      the call.  The number of bytes returned may be more or less than
-      requested.
+      Read on the compressed stream is done to satisfythe call. The
```

Comment:
`satisfythe ` - a typo.

Reply:
Thanks for your code review. I fixed this typo.